README.md

```diff
@@ -102,9 +101,15 @@ export class FooComponent {
 }
 ```
 
-## Customizing request handling
+## Request handling
 
-Starting with Angular@19. The build plugin makes use of `server.ts` file to handle requests. The default Angular scaffolding does generate incompatible code for Netlify so build plugin will swap it for compatible `server.ts` file for you automatically if it detects default one being used. If you need to customize the request handling, you can do so by copying one of code snippets below to your `server.ts` file.
+Starting with Angular@19. The build plugin makes use of `server.ts` file to handle requests. The default Angular scaffolding does generate incompatible code for Netlify so build plugin will swap it for compatible `server.ts` file for you automatically if it detects default one being used. 
```

Comment:
"...makes use of **the** `server.ts` file..." or "...makes use of `server.ts` ~file~ ..."

Reply:
applied in https://github.com/netlify/angular-runtime/pull/223/commits/4a1e5ca844937a17d4dcc7ea4d15f96629cf17d9